app/controllers/quizzes_controller.rb

```diff
@@ -100,6 +106,9 @@ def for_groups
 
   def publish
     @quiz = Quiz.find(params[:id])
+    if @quiz.release_date > Date.today
+      authorize! :manage, @quiz
```

Comment:
oricum nu ar tb sa ai voie sa dai publish daca nu esti autor - trebuie adaugat in afara if-ului oricum